sites/fast-website/src/app/components/side-navigation/side-navigation.ts

```diff
@@ -0,0 +1,48 @@
+import { attr, observable, FASTElement } from "@microsoft/fast-element";
+import {
+    CommunityContentPlacementData,
+    communityContentPlacementData,
+} from "../../data/community.data";
+import { throttle } from "lodash";
```

Comment:
this should be `lodash-es` so taht it can be tree-shaken

Reply:
Thank you. I will update it.